system/doc/efficiency_guide/processes.xmlsrc

```diff
@@ -358,13 +372,12 @@ true
 7> <input>erts_debug:flat_size(element(2, hd(ets:lookup(T, key)))).</input>
 4094</pre>
 
-      <p>When the data has passed through an Ets table,
-      <c>erts_debug:size/1</c> and <c>erts_debug:flat_size/1</c>
-      return the same value. Sharing has been lost.</p>
+    <p>When the data has passed through an Ets table,
+    <c>erts_debug:size/1</c> and <c>erts_debug:flat_size/1</c>
+    return the same value. Sharing has been lost.</p>
 
-      <p>In a future Erlang/OTP release, it might be implemented a
-      way to (optionally) preserve sharing.</p>
-    </section>
+    <p>In a future Erlang/OTP release, it might be implemented a
+    way to (optionally) preserve sharing.</p>
```

Comment:
Should we mention configure option  `--enable-sharing-preserving`?

Reply:
Done.